lib/utils/runOpen.js

```diff
@@ -12,7 +12,11 @@ function runOpen(uri, options, log) {
     openMessage += `: ${options.open}`;
   }
 
-  return open(`${uri}${options.openPage || ''}`, openOptions).catch(() => {
+  const pageUrl = /^(https?:)?\/\//.test(options.openPage)
```

Comment:
I think better use this package https://github.com/sindresorhus/is-absolute-url for check